scripts/teahouse/send_th_invites.py

```diff
@@ -71,11 +76,13 @@ def user_name_from_wikitext(wikitext):
       [[User:FikarummetBot|FikarummetBot]] ([[User talk:FikarummetBot|talk]]) 09:12, 17 October 2016 (UTC) -> FikarummetBot
     """
 
-    return RE_USER_NAME.match(wikitext).group(1)
+    return RE_USER_NAME.match(wikitext).group(2)
```

Comment:
I think we need a try/except block here since (when I quickly test it) I get AttributeError if the input is weirdly formatted. Since we are reading this from a wikipage I suggest adding the robustness of simply skipping any such. Easiest is to simply return `None` on (that) exception and add `inviter_list = filter(None, inviter_list)` later.